streampark-flink/streampark-flink-proxy/src/main/scala/org/apache/streampark/flink/proxy/FlinkShimsProxy.scala

```diff
@@ -88,10 +88,17 @@ object FlinkShimsProxy extends Logger {
 
     SHIMS_CLASS_LOADER_CACHE.getOrElseUpdate(s"${flinkVersion.fullVersion}", {
       // 1) flink/lib
-      val libURL = getFlinkHomeLib(flinkVersion.flinkHome)
-      val shimsUrls = ListBuffer[URL](libURL: _*)
+      def filterLib(filterLib: File): Boolean = {
+        !(filterLib.getName.startsWith("log4j") || filterLib.getName.startsWith("flink-table-planner-loader"))
```

Comment:
Thank you for your feedback，because ExecutorFactory has two implementation: DefaultExecutorFactory(flink-table-planer) and DelegateExecutorFactory(flink-table-planer-loader), spi load 2 servi will conflict, When flink 1.15 running the process is loaded through DelegateExecutorFactory by DefaultExecutorFactory, so exclude flink-table-planer-loader to add flink-table-planer to classpath, this scheme is relatively simple, and later will study whether there can be a better way

Reply:
Thank you for your feedback.Yes, there is a risk in breaking Flink's expected approach, and I'm looking into separating the validation SQL from the submission logic

Reply:
hi @1996fanrui  ,Thank you for your feedback.I revised the implementation again, you can review it again